jsk_arc2017_baxter/euslisp/lib/pick-interface.l

```diff
@@ -130,6 +130,7 @@
     ;; TODO: in-hand verification
     ;; TODO:   target-obj = in-hand object  -> :set_target_cardboard
     ;; TODO:   target-obj != in-hand object -> :return_object
+    (setq target-obj t)
```

Comment:
Now I got why I made this mistake.
I update the PR #2172 with this review and I forgot to test it.
Maybe you are right, and we need to return `t` or `nil` if we make a state-machine with `make-state-machine` function.
I'm so sorry about it.
https://github.com/start-jsk/jsk_apc/pull/2127#discussion_r119985031